### ISSUE TITLE
InternalServerError: cannot drop schema... when creating sequence

### ISSUE THREAD

**Issue by Mulugruntz** [edgedb/edgedb#1225]
- EdgeDB Version: 1-alpha2
- OS Version: Linux 521d2a510951 4.19.76-linuxkit #1 SMP Thu Oct 17 19:31:58 UTC 2019 x86_64 GNU/Linux (the one from the docker image)

Steps to Reproduce:

1. Create a container from `edgedb/edgedb:1-alpha2` docker image
2. Open a terminal (`edgedb -u edgedb -H edgedb`) inside this container
3. Write
```
START TRANSACTION;
CREATE MIGRATION mig1 TO {
    module default {
        scalar type seq_num_t extending sequence;
    }
};
COMMIT MIGRATION mig1;
COMMIT;
```
4. Result should be 

> InternalServerError: cannot drop schema edgedb_b4775f4e-5036-11ea-8560-dbc30fe86934 because other objects depend on it
Hint: This is most likely a bug in EdgeDB. Please consider opening an issue ticket at https://github.com/edgedb/edgedb/issues/new?template=bug_report.md

![image](https://user-images.githubusercontent.com/13810199/74609464-1a978980-5125-11ea-9dda-15c8b72432b9.png)

The error message is not super helpful.
I created the example SDL from https://edgedb.com/ home page `scalar type issue_num_t extending sequence;` and the migration page https://edgedb.com/docs/cheatsheet/migrations#migrations

From my understanding, it should work but maybe I misunderstood the examples, or maybe they're no longer up to date.

**Comment by Mulugruntz** [comment]
Please note that by copy-pasting the example on the migration page as is, it doesn't work and prints the same error message.

**Comment by Fogapod** [comment]
Can't reproduce, what's your current schema?

**Comment by Mulugruntz** [comment]
There's nothing yet. It's a brand new database. I can try to delete the volume and start again. I'll keep you updated :-)

**Comment by Mulugruntz** [comment]
Okay, I did a `docker volume rm <...>` and created a new container. It works fine now.
I restarted it again and it still works. 

So I'm not sure what happened back there and as I removed the volume I cannot investigate anymore either... sorry :).

So I'll just close it as resolved. Maybe it'll pop again for someone else another day. I hope not because I really love EdgeDB so far :).

Keep up the good work and thanks!
